src/main/java/com/googleinterns/zoomtube/utils/CommentUtil.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.users.User;
+import com.googleinterns.zoomtube.data.Comment;
+import java.util.Date;
+import java.util.Optional;
+
+/** Provides methods to create Comment Entities and Comments. */
+public class CommentUtil {
```

Comment:
To enforce that this is a util class please make this final and add a private constructor. This will enforce it is not extendable and non-instantiable.

Reply:
Done!